android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -1034,6 +1034,12 @@ protected void onResume()
     {
       UiUtils.show(mPointChooser);
       mMapButtonsViewModel.setButtonsHidden(true);
+      if (mPointChooserMode == PointChooserMode.NONE)
+      {
+        // BUG: https://github.com/organicmaps/organicmaps/issues/3945
+        Logger.e(TAG, "nativeIsInChoosePositionMode is true but mPointChooserMode is NONE");
```

Comment:
Let's explicitly crash here to find and fix the bug reason.

Reply:
It is pretty obvious that in #3945 the app started with `Framework.nativeIsInChoosePositionMode() &&  mPointChooserMode == PointChooserMode.NONE`. I don't know why. This state is managed somewhere in the native editor.